lib/src/http/content_type.rs

```diff
@@ -104,6 +104,7 @@ impl ContentType {
         "HTML", HTML, is_html => "text", "html" ; "charset=utf-8",
         "Plain", Plain, is_plain => "text", "plain" ; "charset=utf-8",
         "JSON", JSON, is_json => "application", "json",
+        "MsgPack", MsgPack, is_msgpack => "application", "msgpack",
```

Comment:
The reason I used `application/msgpack` is because [the x- prefix is deprecated](https://tools.ietf.org/html/rfc6648) on MIME types. The most recent/most upvoted answer on [this StackOverflow question](http://stackoverflow.com/questions/6888900/mime-type-for-msgpack#25886130) recommends using `application/msgpack` for this reason. However, I've found `application/x-msgpack` and even `bin/msgpack` used in other places on a Google search, and it seems that msgpack [never registered](https://www.iana.org/assignments/media-types/media-types.xhtml) or [decided on](https://github.com/msgpack/msgpack/issues/194) an official MIME type. So, I'll defer to your judgment on which MIME type we should use here.